setup.py

```diff
@@ -52,6 +54,12 @@ def is_requirement(line):
 CHANGELOG = open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')).read()
 VERSION = get_version('pytest_repo_health', '__init__.py')
 
+if sys.argv[-1] == 'tag':
```

Comment:
should this be added to cookiecutters?

Reply:
It should already be in the Django app one, and isn't relevant to IDAs (we usually don't version them).  Should probably be added to the ones for Python packages and XBlocks if not already there, though.